README.md

```diff
@@ -1,17 +1,42 @@
-# Marker
+<h1 align="center">Marker</h1>
+<p align="center">
+    <br/>
+    <img height=200 src="misc/logo.png" alt="Marker logo"/>
+</p>
 
-A project trying to implement a stable linting API for Rust.
+<br/>
 
-## Contributing
+[![Bors enabled](https://bors.tech/images/badge_small.svg)](https://app.bors.tech/repositories/46214)
+![CI Status](https://github.com/rust-marker/marker/actions/workflows/rust_bors.yml/badge.svg)
+
+**A project trying to implement a stable linting API for Rust.**
+
+> **Note**
+>
+> The project is in the very early stages of development, not everything is implemented yet.
+
+## Status
 
-The internal structure of marker and its components is documented in [`docs/internal/`](./docs/internal/).
+What works:
+- A few of example lints, which can be found in the `marker_lints` directory
+- Partial AST representation and API, both of which are in rapid development
+- Prototype `rustc` driver, to run the lints (being developed along with the API)
+
+What doesn't work:
+- API is very much incomplete, but completing it is the main focus of the project right now
+- No integration with `rust-analyzer` or other drivers, only `rustc` as of now
+- ...and a lot more things! Check out [issues](https://github.com/rust-marker/marker/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) if you want to help
```

Comment:
What do you think about this? I think it sounds more optimistic, but at the same time might be too much. I'm currently really hyped for the project and in life :3

```suggestion
What's next:
- The main goal is to complete the AST representation, and start on user tests
- A focus is also set on tooling. Working with marker should be simple and fun for lint developers and users.
- The project is designed to support multiple drivers and unify code analysis for Rust. Additional, drivers like `rust-analyzer` will be investigated.
- ...and a lot more things! Check out the open issues [issues](https://github.com/rust-marker/marker/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc)

Sounds interesting?
* Do you have a lint idea? Help us out by creating a [user story](https://github.com/rust-marker/design/issues/new?assignees=&labels=A-user-story&template=user-story.md&title=)
* Have you found a bug or have suggestions? Please [create an issue](https://github.com/rust-marker/marker/issues/new), any kind of feedback or question is welcome!
* Do you want to help? Even better, check out [open issues](https://github.com/rust-marker/marker/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) or request help.
```